packages/map/src/map/utils.jsx

```diff
@@ -1,6 +1,8 @@
 import { LngLatBounds } from 'mapbox-gl';
 
 
+const DEFAULT_ZOOM_LEVEL = 14;
```

Comment:
move to `constants.jsx` ?

Reply:
not worth for one constant. And it's used only here